api/src/main/java/io/opencensus/trace/EndSpanOptions.java

```diff
@@ -39,9 +38,7 @@
    * @return a new {@code Builder} with default options.
    */
   public static Builder builder() {
-    return new AutoValue_EndSpanOptions.Builder()
-        .setStatus(Status.OK)
-        .setSampleToLocalSpanStore(false);
+    return new AutoValue_EndSpanOptions.Builder().setStatus(null).setSampleToLocalSpanStore(false);
```

Comment:
Why does the status need to be set to null here?

Reply:
Done.